aapp_runner/helper_functions.py

```diff
@@ -25,10 +26,22 @@
 '''
 
 import logging
+import netifaces
 
 LOGGER = logging.getLogger(__name__)
 
 
+def get_local_ips():
+    inet_addrs = [netifaces.ifaddresses(iface).get(netifaces.AF_INET)
+                  for iface in netifaces.interfaces()]
+    ips = []
+    for addr in inet_addrs:
+        if addr is not None:
+            for add in addr:
+                ips.append(add['addr'])
+    return ips
```

Comment:
Could we import the posttroll version instead? https://github.com/pytroll/posttroll/blob/67b8f3d4df27cbb08aaa4baa681acac73d98bd99/posttroll/address_receiver.py#L60-L68

Reply:
Yes, good point!
Optimally this function should be moved into `__init__.py` or a `utils.py` file in Posttroll. Should I make an issue for that do you think?
Now, I would have to do
`from posttroll.address_receiver import get_local_ips`
I mean it is not intuitive that I must import this from the `address_receiver` of Posttroll...